mpi_common.sh

```diff
@@ -24,12 +24,21 @@ function ompi_setup {
     # doesn't load .bashrc/.bash_profile for non-interactive shells.
     # Only load the OFI component so MPI will fail if it cannot be used.
     export MPI_ARGS="-x LD_LIBRARY_PATH --mca mtl ofi"
+    export MPIEXEC_TIMEOUT=1800
 }
 
 function impi_setup {
-    source /opt/intel/compilers_and_libraries/linux/mpi/intel64/bin/mpivars.sh
+    LIBFABRIC_JOB_TYPE=$1
+    if [ "$LIBFABRIC_JOB_TYPE" = "master" ]; then
+        export I_MPI_OFI_LIBRARY_INTERNAL=0
```

Comment:
Is this not redundant, given the -ofi_internal=0 in the next line?

Reply:
I had experience that for parallel cluster's mpivars.sh you need to export this again otherwise it will pick the installer's library. Let me test if it's ok to remove it for tarball's mpivars.sh

Reply:
You are right it's redundant, I will remove this line and update the PR.